components/com_content/tmpl/article/default.php

```diff
@@ -99,7 +99,8 @@
 		echo $this->item->toc;
 	endif; ?>
 	<div itemprop="articleBody" class="com-content-article__body">
-		<?php echo $this->item->text; ?>
+		<?php // echo HTMLHelper::_('InlineEditing.render', $this->item, $this->item->text, 'articletext', null, 'div'); ?>
```

Comment:
WIP? Work In Progress?

Reply:
Yes. Will use this to test the editor field.